src/promptflow/tests/sdk_cli_azure_test/unittests/test_run_operations.py

```diff
@@ -6,6 +6,9 @@
 from promptflow.azure import PFClient
 from promptflow.exceptions import UserErrorException
 
+FLOWS_DIR = "./tests/test_configs/flows"
+DATAS_DIR = "./tests/test_configs/datas"
```

Comment:
Why could these no reference vars could pass the pre commit hooks?

Reply:
I think pre commit hook only checks local vars, this is a global var